File: Realtimemonitoring/Monitoring.Web/Scripts/TS/Incoming/Responce/GenericPushModel.ts
module Main {

    export class GenericPushModel<T> {

        constructor() {

        }

        private _data: Array<T>;

        get Data(): Array<T> {
            return this._data;
        }
        set Data(data: Array<T>) {
            this._data = data;
        }
    }
}  

File: Realtimemonitoring/Src/Monitoring.Web.Contract/Models/Outcoming/Response/GenericPushModel.cs
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace Sychev.Monitoring.Web.Contract.Models.Outcoming.Response
{
    [DataContract]
    public class GenericPushModel<T> where T : class
    {	/// <summary>
        /// Data
        /// </summary> 
        [DataMember]
        public List<T> Data { get; set; }
    }
}